feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/main/StakingViewState.kt

```diff
@@ -259,6 +262,11 @@ class WelcomeViewState(
         ReturnsModel(monthlyEstimation, yearlyEstimation)
     }.asLiveData(scope)
 
+    fun infoActionClicked() {
+        val reward = returns.value
```

Comment:
You can do `val reward = returns.value ?: return` to get rid of nested block on next line

Reply:
Done